XTS/pom.xml

```diff
@@ -17,8 +16,8 @@
     <module>raw-xts-api-demo</module>
     <module>wsat-jta-multi_service</module>
     <module>wsat-jta-multi_hop</module>
-<!--    to enable ssl see https://issues.redhat.com/browse/JBTM-3851-->
-<!--    <module>ssl</module>-->
+    <!--    to enable ssl see https://issues.redhat.com/browse/JBTM-3851-->
```

Comment:
Just for consistency, I would remove the spacing, i.e.
```suggestion
    <!-- to enable ssl see https://issues.redhat.com/browse/JBTM-3851-->
```

Reply:
@jmfinelli I would like to point out that these changes were made by sortpom, not myself, and if I start adding extra changes then that would obfuscate the background to the change.

Reply:
I guess sortpom does not modify comments in the code. That would be up to us. If you prefer to leave things as are, it is fine for me